samples/nested-loops/README.md

```diff
@@ -2,7 +2,8 @@
 This pipeline shows how to define loops and nested loops inside a loop. The nested loops feature demonstrates how Tekton can use the custom task feature to create multiple layers of sub-pipelines and loops on individual pipeline layers.
 
 ## Prerequisites
-- Install [KFP Tekton prerequisites](/samples/README.md) (Kubeflow 1.3+)
+- Install [KFP Tekton 0.9.0+](/guides/kfp_tekton_install.md#standalone-kubeflow-pipelines-with-tekton-backend-deployment)
```

Comment:
For this example it should be fine. Should we mention the recursion since it's still multiple files or mention the parameter to enable/disable it? You konw for the orchestration now seek the method to disable it from me

Reply:
We will try to address the recursion use case at #512 and some future PRs. For this PR we only updating the instructions for parallel loops.

Reply:
OK, I agree.